hnn_core/feed.py

```diff
@@ -74,21 +91,21 @@ def set_prng(self, seed=None):
         self.prng = np.random.RandomState(self.seed)
         if hasattr(self, 'seed2'):
             self.prng2 = np.random.RandomState(self.seed2)
-        # print('ty,seed:',self.ty,self.seed)
+        # print('feed_type,seed:',self.feed_type,self.seed)
 
     def set_event_times(self, inc_evinput=0.0):
         # print('self.p_ext:',self.p_ext)
-        # each of these methods creates self.eventvec for playback
-        if self.ty == 'extpois':
+        # each of these methods creates self.nrn_EventVec for playback
+        if self.feed_type == 'extpois':
```

Comment:
let's do some input validation! If the feed_type is not one of the recognized choices, raise a  ValueError